autofit/aggregator/summary/aggregate_fits.py

```diff
@@ -0,0 +1,130 @@
+import re
+from enum import Enum
+from typing import List
+
+from astropy.io import fits
+from pathlib import Path
+
+from autofit import SearchOutput
+from autofit.aggregator import Aggregator
+
+
+def subplot_filename(subplot: Enum) -> str:
+    subplot_type = subplot.__class__
+    return (
+        re.sub(
+            r"([A-Z])",
+            r"_\1",
+            subplot_type.__name__.replace("FITS", ""),
+        )
+        .lower()
+        .lstrip("_")
+    )
+
+
+class FitFITS(Enum):
```

Comment:
These need to be implemented for other fits files. For example, galaxy_images.fits can be included by defining GalaxyImagesFITS